openvdb_points/tools/AttributeSet.h

```diff
@@ -305,6 +337,7 @@ class AttributeSet::Descriptor
     size_t insert(const std::string& name, const NamePair& typeName);
     NameToPosMap                mNameMap;
     std::vector<NamePair>       mTypes;
+    NameToPosMap                mGroupMap;
```

Comment:
I'm struggling to understand what group "offset" really means here. Can you provide a bit more detail on how per-point group membership is encoded in this scheme?